source/pintab_adafruit_kb2040.py

```diff
@@ -0,0 +1,49 @@
+from board import (
+    A3,
+    A2,
+    A1,
+    A0,
+    SCK,     
+    MISO,
+    MOSI,
+    D10,   
+    NEOPIXEL,
+    SDA,
+    SCL,
+    BUTTON,
+    D9,
+    D8,
+    D7,
+    D6,
+    D5,
+    D4,
+    D3,
+    D2,
+    D1,
+    D0,
+)
+
+pintab = {
+    1: A3,
+    2: A2,
+    3: A1,
+    4: A0,
+    5: SCK,
+    6: MISO,
+    7: MOSI,
+    8: D10,
+    9: NEOPIXEL,
```

Comment:
And this maps the `NEOPIXEL` as GP9. And the json relies on this decleration.
Or is this not the one we are to use as the act led?